server/laundry.py

```diff
@@ -25,14 +27,18 @@ def two_halls(hall_id, hall_id2):
     try:
```

Comment:
Sorry, why does a `two_halls` method exist? Why not have a generic method that can be passed a list of halls as a query parameter and outputs all of them?

Reply:
I'm not actually sure what this is used for. This could be changed to support an arbitrary number of halls, but I expect there would need to be some changes to wherever it is currently being used in the mobile app.

Reply:
I agree, but let's not delete any endpoints for now since that would require a change in the mobile app. I'll make a pull request for a new end point that takes in a list of halls as a query parameter